Azdev does not properly install / copy pylintrc config file.

After installing azdev, I ran:

`(cli-venv) [03:02 PM] Work-Mac:azure-cli tosin$ azdev setup -c . -r ../azure-cli-extensions/`

When I tried to run style checks I get the following:

```
(cli-venv) [03:23 PM] Work-Mac:azure-cli tosin$ azdev style core

===============
| Style Check |
===============

Modules: core

Running flake8 on modules...
Running pylint on modules...


 Results 
=========


/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/_help_loaders.py:16:23: E261 at least two spaces before inline comment
/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/_help_loaders.py:20:1: E302 expected 2 blank lines, found 1
...
...
/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/tests/test_help.py:455:117: E231 missing whitespace after ':'
/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/tests/test_help.py:459:5: E303 too many blank lines (2)
/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/tests/test_help.py:460:5: E301 expected 1 blank line, found 0
1     E203 whitespace before ':'
6     E231 missing whitespace after ':'
3     E261 at least two spaces before inline comment
2     E301 expected 1 blank line, found 0
3     E302 expected 2 blank lines, found 1
2     E303 too many blank lines (2)
2     W291 trailing whitespace

Flake8: FAILED

Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/__main__.py", line 7, in <module>
    pylint.run_pylint()
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/__init__.py", line 19, in run_pylint
    Run(sys.argv[1:])
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/lint.py", line 1348, in __init__
    linter.read_config_file(verbose=self.verbose)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/config.py", line 654, in read_config_file
    raise IOError("The config file {:s} doesn't exist!".format(config_file))
OSError: The config file /Users/tosin/.azdev/env_config/Users/tosin/Repos/cli-venv/config_files/cli_pylintrc doesn't exist!

Pylint: FAILED

```
I got the command to work by doing the following: 

```
(cli-venv) [03:27 PM] Work-Mac:azure-cli tosin$ cp /Users/tosin/Repos/azure-cli-dev-tools/src/azdev/config/cli_pylintrc /Users/tosin/.azdev/env_config/Users/tosin/Repos/cli-venv/config_files/cli_pylintrc
```

```
(cli-venv) [01:00 PM] Work-Mac:azure-cli tosin$ azdev style core

===============
| Style Check |
===============

Modules: core

Running flake8 on modules...
Running pylint on modules...


 Results 
=========


/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/_help.py:182:9: E303 too many blank lines (2)
/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/tests/test_help.py:505:5: E303 too many blank lines (2)
/Users/tosin/Repos/azure-cli/src/azure-cli-core/azure/cli/core/tests/test_help.py:508:1: E305 expected 2 blank lines after class or function definition, found 1
2     E303 too many blank lines (2)
1     E305 expected 2 blank lines after class or function definition, found 1

Flake8: FAILED

Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/__main__.py", line 7, in <module>
    pylint.run_pylint()
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/__init__.py", line 19, in run_pylint
    Run(sys.argv[1:])
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/lint.py", line 1348, in __init__
    linter.read_config_file(verbose=self.verbose)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/pylint/config.py", line 654, in read_config_file
    raise IOError("The config file {:s} doesn't exist!".format(config_file))
OSError: The config file /Users/tosin/.azdev/env_config/Users/tosin/Repos/cli-venv/config_files/cli_pylintrc doesn't exist!

Pylint: FAILED

(cli-venv) [01:00 PM] Work-Mac:azure-cli tosin$ pip list azdev
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
adal (1.2.1)
alabaster (0.7.12)
antlr4-python3-runtime (4.7.2)
apipkg (1.5)
applicationinsights (0.11.7)
argcomplete (1.9.4)
asn1crypto (0.24.0)
astroid (2.1.0)
atomicwrites (1.3.0)
attrs (18.2.0)
azdev (0.0.1)

```

--- a/azdev/operations/pypi.py
+++ b/azdev/operations/pypi.py
@@ -27,8 +27,6 @@ SETUP_PY_NAME = 'setup.py'
 # region verify History Headings
 def check_history(modules=None):
 
-    require_azure_cli()
-
     # TODO: Does not work with extensions
     path_table = get_path_table(include_only=modules)
     selected_modules = list(path_table['core'].items()) + list(path_table['mod'].items())
@@ -91,7 +89,8 @@ def _check_history_headings(mod_path):
 
         first_version_history = all_versions[0]
         actual_version = cmd('python setup.py --version', cwd=mod_path)
-        actual_version = actual_version.result.strip()
+        # command can output warnings as well, so we just want the last line, which should have the version
+        actual_version = actual_version.result.splitlines()[-1].strip()
         if first_version_history != actual_version:
             errors.append("The topmost version in {} does not match version {} defined in setup.py.".format(
                 history_path, actual_version))

--- a/azdev/operations/style.py
+++ b/azdev/operations/style.py
@@ -14,7 +14,7 @@ from knack.util import CLIError, CommandResultItem
 
 from azdev.utilities import (
     display, heading, subheading, py_cmd, get_path_table, EXTENSION_PREFIX,
-    get_env_config_dir)
+    get_env_config_dir, require_azure_cli)
 
 
 logger = get_logger(__name__)
@@ -27,6 +27,12 @@ def check_style(modules=None, pylint=False, pep8=False):
     selected_modules = get_path_table(include_only=modules)
     pep8_result = None
     pylint_result = None
+
+    if pylint:
+        try:
+            require_azure_cli()
+        except CLIError:
+            raise CLIError('usage error: --pylint requires Azure CLI to be installed.')
 
     if not selected_modules:
         raise CLIError('No modules selected.')

--- a/setup.py
+++ b/setup.py
@@ -44,6 +44,7 @@ setup(
         'azdev.config',
         'azdev.operations',
         'azdev.operations.linter',
+        'azdev.operations.linter.rules',
         'azdev.operations.tests',
         'azdev.operations.extensions',
         'azdev.utilities',
@@ -64,7 +65,7 @@ setup(
         ":python_version<'3.0'": ['pylint~=1.9.2'],
         ":python_version>='3.0'": ['pylint~=2.0.0']
     },
-    package_data={'azdev.config': ['*.*']},
+    package_data={'azdev.config': ['*.*', 'cli_pylintrc', 'ext_pylintrc']},
     include_package_data=True,
     entry_points={
         'console_scripts': ['azdev=azdev.__main__:main']

